Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -138,35 +166,14 @@ private enum TouchableSurface
         [SerializeField]
         protected Vector2 bounds = Vector2.zero;
 
-        /// <summary>
-        /// False if no collider is found on validate.
-        /// This is used to avoid the perf cost of a null check with the collider.
-        /// </summary>
-        private bool usesCollider = false;
-
-        /// <summary>
-        /// The collider used by this touchable.
-        /// </summary>
-        [SerializeField]
-        [FormerlySerializedAs("collider")]
-        private Collider touchableCollider;
-
-        protected void OnEnable()
+        protected new void OnValidate()
         {
-            instances.Add(this);
-        }
+            base.OnValidate();
 
-        protected void OnDisable()
-        {
-            instances.Remove(this);
-        }
-
-        protected void OnValidate()
-        {
             Debug.Assert(localForward.magnitude > 0);
             Debug.Assert(localUp.magnitude > 0);
             string hierarchy = gameObject.transform.EnumerateAncestors(true).Aggregate("", (result, next) => next.gameObject.name + "=>" + result);
-            Debug.Assert(Vector3.Angle(localForward, localUp) > 80, $"localForward and localUp not perpendicular for object {hierarchy}. Did you set Local Forward correctly?");
+            Debug.Assert(Vector3.Dot(localForward, localUp) == 0, $"localForward and localUp not perpendicular for object {hierarchy}. Did you set Local Forward correctly?");
```

Comment:
I'd recommend only checking this if the length of the vector is 1, since i often erroneously get this error message as I'm changing the value from -1 to 1